.github/workflows/integration-tests-smoke.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/build-image@3ec816fe2e6dd3f70511e8115d86e296818c58f8 # cosmos_one_off
         with:
           cl_repo: smartcontractkit/chainlink
-          cl_ref: 6b935c6ff7ec12b16b3f644b64080dec29b5a2c4
+          cl_ref: d0bb5e7811b9b590037452f48e4d381998349e54
```

Comment:
this should be switched back to using the input rather than a hard coded input `${{ github.event.inputs.cl_branch_ref }}`

once the corresponding PR in core is merged: https://github.com/smartcontractkit/chainlink/pull/9273